### PR TITLE
49818-standalone import

### PR DESCRIPTION
**Pull request by RoelWKramer**
[AB#49818](https://dev.azure.com/CloudCompetenceCenter/7755cdf4-1859-48c5-b954-fd6c8e46fe52/_workitems/edit/49818)

### CODE REVIEW

File: src/Dockerfile [hunk 1]
```diff
@@ -33,7 +33,7 @@ FROM application as development
 # Install GOB-Core and GOB-Config in development mode. Will be overridden with
 # mounts from the developer filesystem. Changes made to GOB-Core and GOB-Config
 # will be available immediately.
-RUN git clone --depth=1 https://github.com/Amsterdam/GOB-Core.git
+RUN rm -rf GOB-Core && git clone --depth=1 --branch generic-arguments https://github.com/Amsterdam/GOB-Core.git
```
Comment (RoelWKramer) [inline src/Dockerfile:36]: should be branch master, allows easier branch switching when developing.

File: src/gobimport/__main__.py [hunk 2]
```diff
@@ -2,23 +2,54 @@
...
+             "also known as 'entity'."
+    )
+    import_parser.add_argument(
+        "--application",
+        help="The name of the application to import from"
+    )
+    import_parser.add_argument(
+        "--mode",
+        required=False,
+        help="The mode to use. Defaults to update",
```
Comment (roel-devries) [inline src/gobimport/__main__.py:46]:
```suggestion
        help="The mode to use. Defaults to full",
```

File: src/gobimport/__main__.py [hunk 3]
```diff
@@ -2,23 +2,54 @@
...
+        name="import",
+        description="Start an import job for a collection"
+    )
+    import_parser.add_argument(
+        "--catalogue",
+        required=True,
+        help="The name of the data catalogue (example: \"meetbouten\")"
+    )
+    import_parser.add_argument(
+        "--collection",
```
Comment (roel-devries) [inline src/gobimport/__main__.py:35]:
```suggestion
        "--collection",
        required=True
```

File: src/gobimport/__main__.py [hunk 4]
```diff
@@ -2,23 +2,54 @@
...
+        help="The name of the data collection (example: \"metingen\"), "
+             "also known as 'entity'."
+    )
+    import_parser.add_argument(
+        "--application",
+        help="The name of the application to import from"
```
Comment (roel-devries) [inline src/gobimport/__main__.py:41]:
```suggestion
        required=False,
        help="The name of the application to import from"
```

**Review by roel-devries — CHANGES_REQUESTED** [review]

File: src/gobimport/__main__.py [hunk 5]
```diff
@@ -2,23 +2,56 @@
...
+        description="Start an import job for a collection"
+    )
+    import_parser.add_argument(
+        "--catalogue",
+        required=True,
+        help="The name of the data catalogue (example: \"meetbouten\")"
+    )
+    import_parser.add_argument(
+        "--collection",
+        required=True
```
Comment (RoelWKramer) [inline src/gobimport/__main__.py:36]:
```suggestion
        required=True,
```

**Review by roel-devries — APPROVED** [review]

**Pull request merged**